packages/flutter/lib/src/material/carousel.dart

```diff
@@ -400,6 +410,28 @@ class _CarouselViewState extends State<CarouselView> {
         }
         return null;
       });
+    Widget contents = widget.children[index];
+
```

Comment:
Might as well group the related logic together :)

```suggestion

    Widget contents = widget.children[index];
```